clang/bindings/python/clang/cindex.py

```diff
@@ -43,7 +43,7 @@
 Most object information is exposed using properties, when the underlying API
 call is efficient.
 """
-from __future__ import absolute_import, division, print_function
```

Comment:
All of these are already activated by default since Python 3.0. `annotations` is necessary though, for some of the type annotation features used in this PR